test/e2e/density.go

```diff
@@ -182,6 +192,28 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			Logf("Schedling additional Pods to measure startup latencies")
+			var latencies []float64
+			additionalNameStr := strconv.Itoa(minionCount) + "-" + string(util.NewUUID())
+			additionalRCName = "my-hostname-density" + additionalNameStr
+			config = RCConfig{Client: c,
+				Image:         "gcr.io/google_containers/pause:go",
+				Name:          additionalRCName,
+				Namespace:     ns,
+				PollInterval:  itArg.interval,
+				PodStatusFile: fileHndl,
+				Replicas:      minionCount,
+			}
+			latencies, err = RunRC(config)
+			expectNoError(err)
+			sort.Float64s(latencies)
```

Comment:
Why not just return []time.Duration and implement the sort interface for it? (eg: https://github.com/GoogleCloudPlatform/kubernetes/pull/9523/files#diff-71b549d539c828b858c356f26824d75dR45). That would make the logging and collection simpler since you won't have to translate float to time.